src/main/kotlin/calculator/Operator.kt

```diff
@@ -0,0 +1,17 @@
+package calculator
+
+enum class Operator(val remark: String, val compute: (Int, Int) -> Int) {
+    PLUS("+", fun(number1: Int, number2: Int) = number1 + number2),
```

Comment:
타입이 생성자 파라미터인 high order funtion에 명시되어 있는데 아래와 같이 표현하면 어떨까요?

```suggestion
    PLUS("+", { a, b -> a + b }),
```